src/Security/JsonWebTokenAuthenticator.php

```diff
@@ -72,11 +75,8 @@ public function supports(Request $request)
     public function getCredentials(Request $request)
     {
         $authorizationHeader = $request->headers->get('X-JWT-Authorization');
-        if (preg_match('/^Token (.*)$/', $authorizationHeader, $matches)) {
-            return $matches[1];
-        }
-
-        return null;
+        preg_match('/^Token (.*)$/', $authorizationHeader, $matches);
```

Comment:
same here, regexp could  be `/^Token (\S+)$/`

Reply:
fixed!